mod/groups/start.php

```diff
@@ -818,3 +822,67 @@ function groups_members_menu_setup($hook, $type, $menu, $params) {
 
 	return $menu;
 }
+
+/**
+ * Helper handler to correctly resolve page owners on group routes
+ *
+ * @see default_page_owner_handler()
+ * 
+ * @param string $hook   "page_owner"
+ * @param string $type   "system"
+ * @param int    $return Page owner guid
+ * @param array  $params Hook params
+ * @return int|void
+ */
+function groups_default_page_owner_handler($hook, $type, $return, $params) {
```

Comment:
Since groups are very common, let's add a `@see` declaration to `elgg_get_page_owner_guid()` referencing this function. It will help future devs track down this magic.